main/reference/ertp-api/purse.md

```diff
@@ -1,11 +1,12 @@
 # Purse
+
 `Purses` hold digital assets. They are created to hold assets of a single `brand`,
 and their `brand` cannot be changed. For example, you might create a `purse` to
 hold Quatloos. That `purse` can only hold Quatloos; it can't hold Moola or theater
 tickets or anything else. And you can't change that `purse` so that it now holds
-Moola instead; you need to create a `purse` that holds only Moola.
+Moola instead. You would need to create a `purse` that holds only Moola.
 
-Digital assets in `purses` and `payments` can be any of:
+Digital assets in `purses` can be any of:
```

Comment:
why drop `payments` here?

Reply:
Because this topic is discussing purses; including a detail like that about payments isn't very helpful. Such information should instead also appear on the payments topic.  
Upon inspection, I noticed that payments did not have that info, so I added it to the payments topic as well.